src/main/resources/config.sk

```diff
@@ -56,6 +56,10 @@ release channel: @release-channel@
 # 'prerelease' means that development releases will also be checked for.
 # Initial value of this depends on the Skript version you first installed; it was '@release-channel@'.
 
+send reloading info to ops: false
+# Whether Skript should alert operators when someone reloads a script file.
+# This will look like the normal message you receive when reloading a script, containing reload time, errors and/or warnings.
```

Comment:
This isn't really necessary anymore since it's gated behind an opt-in permission

Reply:
You want me to remove the config?

Reply:
yes it's not necessary
otherwise you have to both enable this and give a permission to a player